crates/solver/src/settlement_submission/submitter/common.rs

```diff
@@ -43,7 +43,11 @@ pub async fn submit_raw_transaction(
         .map_err(|err| SubmitApiError::Other(err.into()))?;
 
     let handle = parse_json_rpc_response::<H256>(&body)?;
-    tracing::debug!("transaction handle: {}", handle);
+    tracing::info!(
```

Comment:
Is there a reason to change `tracing::debug!` to `tracing:info!`?
Could this have unexpected side effects, like flooding logs are such?

Reply:
Seems like a replacement change but it is more a moved log from here: 
https://github.com/gnosis/gp-v2-services/blob/f07fc12f1d08bb05b40c3123f0ae4915fbef94a6/crates/solver/src/settlement_submission/submitter.rs#L316
This log contained hash print a few PRs behind, so just wanted to stay consistent with that.

Reply:
Ahh, I see. 👌